tests/test_graphical_units/test_creation.py

```diff
@@ -19,6 +19,15 @@ def construct(self):
         self.play(Uncreate(square))
 
 
+class AddTest(Scene):
+    def construct(self):
+        square = Square()
+        self.play(Add(square))
+
+
+# There is no RemoveTest because it would be all blank pixels.
```

Comment:
That's still a valid test imo

Reply:
I tried and the tester thing complained to me that it wasn't a valid test cause it had blank pixels.. Now that I am thinking about it again, I could add two mobs, and remove one. That should work.

Reply:
that's strange. The tester _shouldn't_ make this.

Reply:
I added this test a while back